crates/workspace/src/settings/line_ending.rs

```diff
@@ -0,0 +1,31 @@
+use std::fmt;
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Default, serde::Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum LineEnding {
```

Comment:
I think it is very reasonable / inevitable that we end up with our own copies of these settings in addition to the ones from biome.

- For each one we need a `Deserialize` method that works nicely with the toml crate and uses kebab-case
- For some, like here, we add additional user facing options like `Auto` that don't make it into the underlying formatter option
- For some, like `IndentWidth`, we may choose to have our own `IndentWidth::default()` that is different from what biome's is